chain/src/chain.rs

```diff
@@ -656,6 +656,7 @@ impl<CS: ChainStore + 'static> ChainService<CS> {
                     }
                 } else {
                     *verified = Some(false);
+                    return Err(found_error.unwrap().into());
```

Comment:
We wanna mark all its descendants `block_ext.verified = Some(false)` when verification failed, so here should not return but continue to mark.